frontend/src/app/model/Transaction.ts

```diff
@@ -1,10 +1,10 @@
 export default interface Transaction {
   id: number;
+  bankId: number;
+  transactionTitle: string;
   accountNumber: string;
-  direction: string;
+  direction: 'PAY' | 'RECEIVE';
```

Comment:
You could create a global file like `consts.ts` for names/enums like these. Hardcoding is very error prone.

Reply:
Well, it's a definition of type, so I don't thing that any const container is a good place to store it. 
What I could do is to define sth like:
`type TransactionDirection = 'PAY' | 'RECEIVE';` in the same file. I tried TS enum and doesn't seem very handy here.

Reply:
Well, how about making direction a separate unmutable type? Something like `Object.freeze()`. Using strings to define type like that seems just unreasonable.

Reply:
Must